src/js/app.js

```diff
@@ -22,5 +22,17 @@ $(() => {
   services.instantsearch.init();
   services.highlightjsBadge.init();
   services.progressbar.init();
-  import(/* webpackPrefetch: true */ './experiments'); // imports all experiments
+
+  const isGatedPath =
+    ['/docs/', '/docs/2.0/'].includes(window.location.pathname) ||
+    ['-preview/', 'view/2.0/'].includes(window.location.pathname.slice?.(-9)) ||
+    /language-(javascript|python)/gm.test(window.location.pathname);
+  if (!isGatedPath) {
+    site.toc.highlightTocOnScrollOnce();
+  }
```

Comment:
could we add a quick comment here on the purpose of this?

Reply:
done.